src/Pages/form/Components/ArrivalOnset.tsx

```diff
@@ -0,0 +1,408 @@
+                  setShowPickerClear(false);
+                  setShowPickerUnknown(true);
+                }}
+              >
+                <Box>
+                  <Typography variant="subtitle1">Unknown onset</Typography>
+                </Box>
+              </ToggleButton>
+            </OnsetStyledToggleButtonGroup>
+          </Box>
+        </Grid>
+        <Box>
+          {showClearPicker ? (
+            <Box className={classes.boxOnsetPicker}>
+              <Grid container spacing={7}>
+                <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                  <Grid item>
+                    <KeyboardDatePicker
+                      variant="inline"
+                      format="dd/MM/yyyy"
+                      label="Select date"
+                      autoOk={true}
+                      value={clearDate}
+                      onChange={(date: any) => setClearDate(date)}
+                      KeyboardButtonProps={{
+                        "aria-label": "change date",
+                      }}
+                      className={classes.addMarginTop}
+                    />
+                  </Grid>
+                  <Grid item>
+                    <KeyboardTimePicker
+                      variant="inline"
+                      label="Select time"
+                      value={clearDate}
+                      autoOk={true}
+                      onChange={(date: MaterialUiPickersDate) => {
+                        setClearDate(date);
+                      }}
+                      KeyboardButtonProps={{
+                        "aria-label": "change time",
+                      }}
+                      className={classes.addMarginTop}
+                    />
+                  </Grid>
+                  <Grid item>
+                    <Typography>Duration</Typography>
+                    <Box
+                      className={classes.boxDuration}
+                      style={{
+                        backgroundColor: `${
+                          ArrivalClearDiff !== "" ? "#6ED0BB" : "#C4C4C4"
+                        }`,
+                      }}
+                    >
+                      {ArrivalClearDiff !== "" ? (
+                        <Typography
+                          style={{
+                            color: `${
+                              ArrivalClearDiff !== "" ? "#FFFFFF" : "#000000"
+                            }`,
+                          }}
+                        >
+                          {ArrivalClearDiff.days} : {ArrivalClearDiff.hours} :{" "}
+                          {ArrivalClearDiff.minutes}
+                        </Typography>
+                      ) : (
+                        "dd:hh:mm"
+                      )}
+                    </Box>
+                  </Grid>
+                </MuiPickersUtilsProvider>
+              </Grid>
+            </Box>
+          ) : null}
+        </Box>
+        <Box>
+          {showUnknownPicker ? (
```

Comment:
If you want to check the condition to display only one component you can use `showUnknownPicker && (...)`.So, please remove the false conditions.